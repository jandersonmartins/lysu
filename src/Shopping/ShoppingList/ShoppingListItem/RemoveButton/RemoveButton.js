import React from 'react'
import * as S from './RemoveButton.styles'

const RemoveButton = props => (
  <S.RemoveButton {...props}>
    Remover
  </S.RemoveButton>
)

export default RemoveButton
