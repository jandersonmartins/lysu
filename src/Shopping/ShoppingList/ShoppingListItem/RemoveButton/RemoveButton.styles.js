import styled from 'styled-components'

const RemoveButton = styled.button`
  border: none;
  background: #CA3433;
  color: ${props => props.theme.colors.white};
  font-weight: 700;
  height: 3rem;
`

export { RemoveButton }
