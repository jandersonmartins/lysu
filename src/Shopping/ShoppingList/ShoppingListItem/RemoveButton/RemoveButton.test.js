import React from 'react'
import { render } from '@testing-library/react'
import { ThemeProvider } from 'styled-components'
import RemoveButton from './RemoveButton'

test('should render correctly', () => {
  const { container } = render(
    <ThemeProvider theme={{ colors: {} }} >
      <RemoveButton />
    </ThemeProvider>
  )

  expect(container.firstChild).toMatchSnapshot()
})
