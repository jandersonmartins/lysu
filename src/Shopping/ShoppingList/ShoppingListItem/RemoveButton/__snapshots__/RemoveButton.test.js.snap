// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
.c0 {
  border: none;
  background: #CA3433;
  font-weight: 700;
  height: 3rem;
}

<button
  class="c0"
>
  Remover
</button>
`;
